app/src/main/AndroidManifest.xml

```diff
@@ -17,6 +20,19 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name=".ChatLogActivity">
```

Comment:
Can you change this to MainActivity and add ChatLogActivity below with the rest of activities please? (discussed internally)